apps/frontend/src/app/PageSettings/DatabaseCard.tsx

```diff
@@ -20,6 +31,13 @@ import { useBoolState } from '@genshin-optimizer/react-util'
 import { range } from '../Util/Util'
 import UploadCard from './UploadCard'
 
+const CLIENT_ID =
```

Comment:
Please add these values as envvars to https://github.com/frzyc/genshin-optimizer/blob/master/apps/frontend/.env.

Reply:
done